Formula/xcbuild.rb

```diff
@@ -0,0 +1,24 @@
+class Xcbuild < Formula
+  desc "Xcode-compatible build tool"
+  homepage "https://github.com/facebook/xcbuild"
+  url "https://github.com/facebook/xcbuild.git",
+    :tag => "0.1.0",
+    :revision => "08575f65f1c907e280ed4663b5461b7cfaf5cfaa",
+    :shallow => false
+  head "https://github.com/facebook/xcbuild.git",
+    :shallow => false
```

Comment:
more of those nebulous and esoteric white space rules whose secrets are never truly known in all their splendor :)